test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -130,7 +131,10 @@ def hook(context):
         if not os.path.exists(directory):
             os.makedirs(directory)
 
-        pb.savev(os.path.join(directory, filename), "png", [], [])
+        try:
+            pb.savev(os.path.join(directory, filename), "png", [], [])
+        except:
+            pass
```

Comment:
why is that upload failing, and if it happens the developer will be puzzled by not seeing the screenshot.
I think we should not just cover the issue but at lease report that the upload failed

Reply:
For the screenshot, I will create the next PR.